module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  POST, GET, PUT, DELETE
+
 2. What is Sinatra?
+  A lightweight version of Ruby on Rails.
+  
 4. What is MVC?
+  Model View Controller
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  Consistency throughout the code.  Makes for easier debugging/troubleshooting.
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  Instance, Local, and Class variables.
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed`?
+    ```ruby
+  get '/horses' do
+    name = 'Mr. Ed'
+    @count = 1
+    erb :index

```

Comment:
in order for `name` to be available in the `index.erb` view template, we would need to replace line 36 with the following `erb :index, locals: {name: name}`